rest/utilities_testing_blip_client.go

```diff
@@ -1587,6 +1621,15 @@ func (btc *BlipTesterCollectionClient) upsertDoc(docID string, parentVersion *Do
 	return &rev, nil
 }
 
+// Delete creates a tombstone for the document.
+func (btc *BlipTesterCollectionClient) Delete(docID string, parentVersion *DocVersion) (DocVersion, error) {
```

Comment:
should this require that parentVersion != nil? While you can create a tombstone on sync gateway I don't think this is a valid thing for lite to do.

Reply:
Yep, valid point - might as well add this guard to prevent misuse.